app/src/main/res/drawable/create_box.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Foarte nice. Incearca sa folosesti un state list drawable pentru a schimba culorile in functie de starile butonului (pressed versus non-pressed) pentru un efect vizual mult mai puternic.